packages/odyssey/CHANGELOG.md

```diff
@@ -1,17 +1,32 @@
-<!-- CHANGELOG Example
+# Changelog
 
-# 0.0.1
+All notable changes to this project will be documented in this file.
 
-### Features
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-- [`somePullRequest`](https://github.com/okta/odyssey/pulls/) - Description of the new feature.
+## [Unreleased]
+- Rebrand type and color changes
 
-### Fixes
+## [0.1.0] - 2020-04-01
 
-- [`somePullRequest`](https://github.com/okta/odyssey/pulls/) - Description of the problem here and how it was solved.
+### Added
+- Table: styling and documentation for default table behavior, sorting, and various content types
+- Select: support for default select input behavior via plain HTML and Choices.js
+- Styled Element Glossary: blockquote, cite, del, em, ins, mark, s, small, sub, sup, strong
+- Component Status Page
+- Code of Conduct
 
-### Other
+### Changed
```

Comment:
Going to mark them inline since we could have breaking changes result from additions or removals too:

<img width="464" alt="Screenshot 2020-04-03 16 46 42" src="https://user-images.githubusercontent.com/36284167/78413091-b80e2600-75ca-11ea-9648-2fac897deb1c.png">

Reply:
Nice! Good idea.